napari/layers/shapes/shape_models/shape.py

```diff
@@ -1,33 +1,38 @@
 from abc import ABC, abstractmethod
-import numpy as np
 from copy import copy
-from vispy.color import Color
+
+import numpy as np
+
 from ..shape_utils import (
     triangulate_edge,
     triangulate_face,
     is_collinear,
     poly_to_mask,
     path_to_mask,
 )
+from ....utils.colormaps.standardize_color import (
+    hex_to_name,
+    rgb_to_hex,
+)
 
 
 class Shape(ABC):
-    """Base class for a single shape
+    """Base class for a single shape.
 
     Parameters
     ----------
     data : (N, D) array
         Vertices specifying the shape.
     edge_width : float
         thickness of lines and edges.
-    edge_color : str | tuple
-        If string can be any color name recognized by vispy or hex value if
-        starting with `#`. If array-like must be 1-dimensional array with 3 or
-        4 elements.
-    face_color : str | tuple
-        If string can be any color name recognized by vispy or hex value if
-        starting with `#`. If array-like must be 1-dimensional array with 3 or
-        4 elements.
+    edge_color : str or array-like
+        A single color for all of the shape marker borders. A string can be
+        the color name or its hex values (starting with `#`). Array-like should
+        have 3 or 4 elements in a single row (can be 2D) in a RGB(A) format.
+    face_color : str or array-like
+        Color of the shape marker body. A string can be the color name or its
+        hex value (starting with `#`). Array-like should have 3 or 4 elements
+        in a single row (can be 2D) in a RGB(A) format.
```

Comment:
@sofroniewn Do you like this phrasing better than the previous one?

Reply:
Much better, thanks! You can propagate that to the rest of the individual shapes